katrain/core/sgf_parser.py

```diff
@@ -1,4 +1,5 @@
 import copy
+import chardet
```

Comment:
this is an external package, right? please add it to setup.py/requirements.txt

Reply:
Uploaded a new patch adding it to setup.py (not sure I added it the right way, though). My local "pip install ." works.
It's already listed in requirements.txt.